scripts/setup-environment.sh

```
@@ -0,0 +1,150 @@
+  fi
+  info "Created .env – remember to add your secrets."
+fi
+
+# 3) Interactive secrets prompt ----------------------------------------------
+if [[ "$NON_INTERACTIVE" == false ]]; then
+  printf "\n--- Environment configuration ---\n"
+
+  read -rp "GitHub Personal Access Token [press Enter to skip]: " GITHUB_TOKEN
+  read -rp "OpenAI API Key [press Enter to skip]: " OPENAI_API_KEY
+
+  printf "\nAI review provider determines which bot(s) comment on pull requests:\n  1) cursor     – internal GPT-4 reviewer using your OpenAI key\n  2) coderabbit – external SaaS reviewer (no token cost, code sent to CodeRabbit)\n  3) both       – run both for maximum coverage\n  4) skip       – configure later\n"\n\n  PS3="Enter choice [1-4]: "
```

Shellcheck (0.10.0)

[error] This printf format string has no variables. Other arguments are ignored.

(SC2182)